bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/GuideGroup.scala

```diff
@@ -236,19 +242,23 @@ case class GuideGroup(grp: GuideGrp) extends java.lang.Iterable[GuideProbeTarget
     getTargets.toList.iterator
 
   def getParamSet(f: PioFactory): ParamSet = {
-    import GuideGroup.{ParamSetName, AutoInitialTag, AutoActiveTag, ManualTag }
+    import GuideGroup.{grp => _, _}
```

Comment:
In this case, I did it because the `GuideGroup.grp` lens would otherwise clash with the `grp` field of class `GuideGroup`.